<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QWidget" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgba(163, 224, 255, 1);
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 11pt;
}
</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2980b9;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 6px;
    padding: 6px 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2980b9;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textNote">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdressLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CompanyLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Full Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Email :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Phone Number :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Adress :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Country :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Company :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string> Client type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(0, 68, 204, 1);
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Notes :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCountry">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>330</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>480</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QTextEdit, QComboBox {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
   color: white;
    background-color: #3498db;
    border-radius: 6px;
    font-weight: bold;
}
QToolButton:hover{
  background-color: #2980b9;
}</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
